### DISCUSSION TITLE
personal-development-shahrivar-1402/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#21] [Announcements]
# personal-development-shahrivar-1402/

بنا به روالی که از بهمن پارسال شروع کردم، گزارش ماهانه شهریور ۱۴۰۲ رو اینجا می‌گذارم.
این ماه نسبتاً بیشتر از بقیه ماه‌ها با آدم‌ها ارتباط داشتم. از خانواده خودم و همسرم تا دوستانم که الان دیگه تقریباً همه اونهایی که تو دوره ارشد با هم بودیم، مهاجرت کردند.
حس عجیبی هست که اونهایی که تا همین چند هفته پیش با هم می‌نشستیم و حرف می‌زدیم الان یه جای دیگه دنیا هستند.

https://aprd.ir/personal-development-shahrivar-1402/

**Comment by hiwasham** [comment]
سلام امیر جان
- من تقریبا 13-14 سال پیش کلاس دکتر نوابی بودم. تجربه خاصی بود. دیدن آدمهایی که اولاً در دنیای بیرون نمی بینی شون، دوماً ویژگی های عجیب غریبی دارن که آدم رو غافلگیر میکنه. دکتر نوابی، دکتر فخرایی (که چند سال پیش فوت کرد) و تعدادی از استادهایی که اونجا داشتیم برای من جز این آدمها بودن. 
- دوره ای که من بهت داده بودم فعلاً حالت آزمایشی داره. فعلاً درگیر محتواش هستم بعدا روی فرآیند و اجراش کار میکنم به ویژه موضوعی که گفتی. اینکه چطوری اجراش کنیم و مراقب باشیم که تبدیل به یه بار و اجبار نشه.
